api/v1alpha1/tenantcontrolplane_types.go

```diff
@@ -96,11 +96,24 @@ type AddonSpec struct {
 	Enabled *bool `json:"enabled,omitempty"`
 }
 
+type KonnectivitySpec struct {
+	// +kubebuilder:default=true
```

Comment:
Define a new field for a dynamic konnectivity agent and server container image, sticking to the default value of `v0.0.16`, since it's the version that we tested and it's working.